Detections/OfficeActivity/Office_MailForwarding.yaml

```diff
@@ -20,17 +20,17 @@ relevantTechniques:
   - T1020
 query: |
 
-  let timeframe = 7d
+  let timeframe = 7d;
   OfficeActivity
   | where TimeGenerated >= ago(timeframe)
   | where Operation =~ "Set-Mailbox"
   | where Parameters contains "ForwardingSmtpAddress"
   | extend parsed = parse_json(Parameters)
   | extend parameterName = parsed[1].Name, fwdingDestination = tostring(parsed[1].Value)
   | where isnotempty(fwdingDestination)
-  | summarize TimeGenerated=max(TimeGenerated) by UserId, fwdingDestination
-  | summarize TimeGenerated=max(TimeGenerated), userCount = dcount(UserId), UserId = makelist(UserId) by fwdingDestination
+  | summarize TimeGenerated=max(TimeGenerated) by UserId, ClientIP, fwdingDestination
```

Comment:
not sure we want to do this: this will explode the number of results if there are many IPs?

Reply:
Tested on more than one workspace with varied data, we should be good here.